orchestration/scripts/post-ingest-file.py

```diff
@@ -0,0 +1,22 @@
+import google.auth
```

Comment:
Nitpick on the file name: For some reason my brain saw "post" and thought "post-processing", then got confused when it saw this was launching a job. Could we rename the script to something like `launch-file-ingest` or `submit-file-ingest`?

Reply:
sure thang